app/Modules/Auth/AuthController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Modules\Auth;
+
+use Tempest\Auth\Authenticator;
+use Tempest\Auth\CredentialsResolver;
+use Tempest\Auth\DatabaseAuthenticationCall;
+use Tempest\Auth\Exceptions\MissingIdentifiableException;
+use Tempest\Container\Tag;
+use Tempest\Http\Get;
+use Tempest\Http\Post;
+use Tempest\Http\Request;
+use Tempest\Http\Response;
+use function Tempest\response;
+
+final readonly class AuthController
+{
+    public function __construct(
+        #[Tag('database')]
+        private CredentialsResolver $credentialsResolver,
+        #[Tag('database')]
+        private Authenticator $authenticator,
+    ) {
+    }
+
+    #[Post('/login')]
```

Comment:
Shouldn't we have Get routes as well?

Reply:
To show the login and logout form*

Reply:
FYI I can add these after this is merge, just wanted to check. Forms are still a mess, so I don't expect you to struggle with those :p

Reply:
Got it!
I won't add these for now then.